packages/commonwealth/server/routes/updateChain.ts

```diff
@@ -118,6 +119,52 @@ const updateChain = async (
     return next(new AppError(Errors.InvalidTerms));
   }
 
+  const snapshotSpaces = await models.CommunitySnapshotSpaces.findAll({
+    where: { chain_id: chain.id },
+    include: {
+      model: models.SnapshotSpace,
+      as: 'snapshot_space',
+    },
+  });
+
+  // Check if any snapshot spaces are being removed
+  const removedSpaces = snapshotSpaces.filter((space) => {
+    return !snapshot.includes((space as any).snapshot_space.snapshot_space);
+  });
+  const existingSpaces = snapshotSpaces.filter((space) => {
+    return snapshot.includes((space as any).snapshot_space.snapshot_space);
+  });
+  const existingSpaceNames = existingSpaces.map((space) => {
+    return (space as any).snapshot_space.snapshot_space;
+  });
+
+  for (const spaceName of snapshot) {
+    // check if its in the mapping
+    if (existingSpaceNames.includes(spaceName)) {
+      continue;
```

Comment:
Don't use continue -- just reverse the boolean logic so there's no `else` block.